app/models/druid_version_zip.rb

```diff
@@ -118,6 +133,11 @@ def fetch_zip_version
     raise 'No version info matched from `zip -v` ouptut'
   end
 
+  # @return [String] the option included with "zip -s"
+  def zip_split_size
+    '10g'
```

Comment:
should this be a config setting?  

if it changes, does it impact stuff already replicated being retrieved and reassembled?

Also, should the constant in preserved_copy.replicate! be derived from the value here?

Reply:
According to Julian, it sounds like no.  If it becomes necessary, it is simple enough to do later.  Changing the value affects the overall logic not at all, because reassembly (unimplemented, repeatedly declared out of scope) relies only on the *count* of parts, not their size.  Once you have all the enumerated parts, you can unzip them, regardless of their size.

This is specifically an option string passed to `zip`.  While related to the enqueue constraint, that is an Integer that must anticipate overhead from zip internal structure.  The constraint and the option operate in different phases of asynchronous operation, so I'm not interested in making them tightly coupled.

Reply:
Having gotten frustrated spelunking for "where are the hidden hardcoded assumptions" in moab-versioning ... is there a way to make this particular constant more obvious?

Reply:
I think having its own overridable method makes it much more highly visible.